renovate.json5

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":gitSignOff",
+    "helpers:pinGitHubActionDigests"
+  ],
+  // This ensures that the gitAuthor and gitSignOff fields match
+  "gitAuthor": "renovate[bot] <bot@renovateapp.com>",
+  "includePaths": [
+    ".github/workflows/**",
+    "Dockerfile",
+    "go.mod",
+    "go.sum",
+    "Makefile",
+  ],
+  postUpdateOptions: [
+    "gomodTidy"
+  ],
+  "ignoreDeps": [
+    "github.com/cilium/cilium"
+  ],
+  "pinDigests": true,
+  "ignorePresets": [":prHourlyLimit2"],
+  "separateMajorMinor": true,
+  "separateMultipleMajor": true,
+  "separateMinorPatch": true,
+  "pruneStaleBranches": true,
+  "baseBranches": [
+    "master",
+    "v0.11"
```

Comment:
Should we add anything about this in `RELEASE.md`? I'm assuming we will want to `s/v0.11/v0.12/` when cutting the first `v0.12` release.

Reply:
Good point, I'll take a look at updating docs.